tests/suites/test_suite_pk.function

```diff
@@ -417,26 +417,30 @@ exit:
 void pk_rsa_overflow( )
 {
     mbedtls_pk_context pk;
-    size_t hash_len = SIZE_MAX;
+    size_t hash_len = SIZE_MAX, sig_len = SIZE_MAX;
+    unsigned char hash[50], sig[100];
```

Comment:
Where do the numbers `50` and `100` come from? Add a comment explaining the rationale for their sizes, in case anybody wants to come back later and change them. If the sizes are constant and nobody would ever change them, consider making well named constants for the sizes to improve readability.

Reply:
These sizes have no true meaning as this test checks for hash length overflows and these buffers are never actually used.